### ISSUE TITLE
construct! macro does not support enum variants with full path

### ISSUE THREAD

**Issue by msrd0** [pacak/bpaf#16]
```rust
::bpaf::construct!(my_crate::MyEnum::Variant { foo, bar })
```

gives the error message

```
error: no rules expected the token `prepare`
...
   = note: this error originates in the macro `$crate::construct` (in Nightly builds, run with -Z macro-backtrace for more info)
```

and prefixing with `::` doesn't even parse

```rust
::bpaf::construct!(::my_crate::MyEnum::Variant { foo, bar })
```

gives the error message

```
error: no rules expected the token `::`
...
```

**Comment by pacak** [comment]
Released 0.4.7, should be working now.

**Comment by msrd0** [comment]
Thanks :heart_eyes_cat:
